presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixOutputTableHandle.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.phoenix;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.prestosql.plugin.jdbc.JdbcOutputTableHandle;
+import io.prestosql.spi.type.Type;
+
+import java.util.List;
+import java.util.Optional;
+
+public class PhoenixOutputTableHandle
+        extends JdbcOutputTableHandle
+{
+    private final boolean hasUUIDRowkey;
```

Comment:
Nit: "row key" seems to be two words per the Phoenix docs

Also, for our code style, we only capitalize the first letter of acronyms, so `HTTP` -> `Http`, `URL` -> `Url`, etc. This makes it easier to read when other words follow acronyms, or you have two acronyms in a row.

So this should probably be `hasUuidRowKey`